lib/hal/video.c

```diff
@@ -198,6 +198,10 @@ BOOLEAN XVideoListModes(VIDEO_MODE *vm, int bpp, int refresh, void **p)
 	
 	DWORD dwAdapter = dwEnc & 0x000000FF;
 	DWORD dwStandard = dwEnc & 0x0000FF00;
+
+	ULONG eepromVidSize, eepromValueType, eepromReturnLength;
+
+	ExQueryNonVolatileSetting(0x08, &eepromValueType, (void*)&eepromVidSize, 4, &eepromReturnLength);
```

Comment:
We should be using named constants for the first parameter. If we don't have any documented, you should craft your own and keep it in this file, just above this line (to make code more descriptive).

Reply:
This was solved by removing this line in its entirety, the same data is being fetched by "`XVideoGetEncoderSettings()`".